docs/tutorial/mount-block-device.rst

```diff
@@ -96,6 +96,26 @@ For the sake of simplicity, we are using admin keys in this example.
 The files are located at the paths shown above on any MicroCeph node.
 Moving forward, we will assume that these files are located at mentioned path.
 
+After 'ceph-common' package installation, 'ceph' command will start looking at the '/etc/ceph/' directory for keyring and conf files, which is not the correct place. 
+This could cause CLI errors similar to this: 
+
+.. code-block:: none
+
+   Error initializing cluster client: ObjectNotFound('RADOS object not found (error calling conf_read_file)')
+ 
+To fix this, 'microceph.ceph' command can be used instead of 'ceph'. To continue using 'ceph' commands as usual, keyring and conf files must be linked to the '/etc/ceph/' directory as follows:
+
+.. code-block:: none
+
+   $ sudo ln -s /var/snap/microceph/current/conf/ceph.conf /etc/ceph/
+   $ sudo ln -s /var/snap/microceph/current/conf/ceph.keyring /etc/ceph/
```

Comment:
These 2 could be combined to:
`$ sudo ln -s /var/snap/microceph/current/conf/ /etc/ceph/`

Reply:
Also look at line 126, 127. These would have to be removed as we are explicitly providing the conf paths.